nautobot/docs/user-guides/graphql.md

```diff
@@ -100,21 +100,21 @@ GraphQL reduces the complexity of performing multiple API calls and correlating
 
 In Nautobot, there is a link to the GraphQL web interface at the bottom right-hand side of the page. The GraphQL web interface is called GraphiQL. Navigating to the URI (`/graphql`), brings up the GraphiQL tool for creating queries. This interface is useful for exploring the possibilities of GraphQL and validating that written queries execute successfully.
 
-![GraphiQL Interface](/user-guides/images/graphql/00-graphiql.png)
+![GraphiQL Interface](./images/graphql/00-graphiql.png)
```

Comment:
Looking at https://nautobot--296.org.readthedocs.build/en/296/user-guides/graphql/, this image works, but the ones below (those using literal `<img>` elements) are broken. Possibly try changing the relative paths below from `./` to `../`?

Reply:
Thanks! Good catch. I just tested this locally.